### PR TITLE
forgot to derive Debug

### PR DESCRIPTION
**Pull request by yoshuawuyts**

### CODE REVIEW

File: src/iterator.rs [hunk 1]
```diff
@@ -11,6 +11,7 @@ use super::*;
 use std::iter;
 
 /// Iterator over a flat-tree.
+#[derive(Debug)]
```
Comment (ralphtheninja) [inline src/iterator.rs]: What does this do?

Comment (yoshuawuyts) [reply src/iterator.rs]: It allows us to do ` println!("{:?}", value)` on the struct. Unless we do embedded programming, all structs should generally have `Debug` on them!

Comment (ralphtheninja) [reply src/iterator.rs]: What happens if it's not used? Compile error?

Comment (yoshuawuyts) [reply src/iterator.rs]: you can't print the struct to stdout. Also can't print any structs that use this struct to stdout. You really want to be able to when debugging, haha.

**Pull request merged**